mempool/docker-compose.yml

```diff
@@ -16,18 +16,22 @@ services:
     environment:
       FRONTEND_HTTP_PORT: $APP_MEMPOOL_PORT
       BACKEND_MAINNET_HTTP_HOST: $APP_MEMPOOL_API_IP
+      NGINX_PORT: $APP_MEMPOOL_PORT
+      NGINX_HOSTNAME: $APP_MEMPOOL_API_IP
+      LIGHTNING_DETECTED_PORT: $APP_LIGHTNING_NODE_PORT
     networks:
       default:
         ipv4_address: $APP_MEMPOOL_IP
   api:
-    image: mempool/backend:v2.4.0@sha256:6036cc6866b782efadc765e8189958940202b3fa1770eb35c35210c86efd2f47
+    image: mempool/backend:v2.5.0@sha256:c58996a3bfd382cb782da5d93e4b8f65f9c02c7a6bb10df4dda0bdf52ba8ed88
     user: "1000:1000"
     init: true
     restart: on-failure
     stop_grace_period: 1m
     command: "./wait-for-it.sh mariadb:3306 --timeout=720 --strict -- ./start.sh"
     volumes:
       - ${APP_DATA_DIR}/data:/backend/cache
+      - ${UMBREL_ROOT}/app-data/lightning/data/lnd:/lnd:ro
```

Comment:
This will be correctly handled by `rsync` here: https://github.com/getumbrel/umbrel/blob/f60bebecae30de37a41633514e339be7ea1914c0/scripts/app#L409 assuming [`data/lnd`](https://github.com/getumbrel/umbrel-apps/tree/112169469178b9e7a106f66c7d32826a07a4bcd6/lightning/data/lnd) exists in the app repo template of the version of Lightning Node app that's being installed.